src/timer.c

```diff
@@ -130,6 +130,11 @@ uint64_t uv_timer_get_repeat(const uv_timer_t* handle) {
 }
 
 
+uint64_t uv_timer_get_due_in(const uv_timer_t* handle) {
+  return handle->timeout - handle->loop->time;
```

Comment:
I think this still needs a check for overflow.

Reply:
I'm not sure what you're asking for, is it this?
```c
uint64_t uv_timer_get_due_in(const uv_timer_t* handle) {
  uint64_t due_in = handle->timeout - handle->loop->time;
  return due_in >= 0 ? due_in : 0;
}
```

Reply:
And do I need to check for `NULL` anywhere?

Reply:
I guess you want me to check that time is not larger than timeout because it will overflow. Pushed that change.